sync/handlers.py

```diff
@@ -181,9 +181,17 @@ def __call__(self, git_gecko, git_wpt, body):
 
 
 class TaskHandler(Handler):
+    """Handler for the task associated with a decision task completing.
+
+    Note that this is using the Treeherder event type at
```

Comment:
@jgraham I just realized that the filter in listen.py must be wrong too: https://github.com/mozilla/wpt-sync/blob/master/sync/listen.py#L194

How is it even working at all? I think we want it to check that `body["state"]` is "completed".